api/infrastructure/parallelcluster-api.yaml

```diff
@@ -142,6 +149,412 @@ Resources:
+        - Action:
+          - route53:ChangeResourceRecordSets
+          - route53:ChangeTagsForResource
+          - route53:CreateHostedZone
+          - route53:DeleteHostedZone
+          - route53:GetChange
+          - route53:GetHostedZone
+          - route53:ListResourceRecordSets
+          - route53:ListQueryLoggingConfigs
+          Resource: !Sub arn:${AWS::Partition}:route53:::hostedzone/*
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: Route53HostedZones
+        - Action:
+          - cloudformation:DescribeStackEvents
+          - cloudformation:DescribeStackResource
+          - cloudformation:DescribeStackResources
+          - cloudformation:DescribeStacks
+          - cloudformation:ListStacks
+          - cloudformation:GetTemplate
+          - cloudformation:CreateStack
+          - cloudformation:DeleteStack
+          - cloudformation:UpdateStack
+          Resource: !Sub
+          - arn:${AWS::Partition}:cloudformation:${Region}:${AWS::AccountId}:stack/*
+          - { Region: !If [IsMultiRegion, '*', !Ref AWS::Region] }
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: CloudFormation
+        - Action:
+          - sns:ListTopics
+          - sns:GetTopicAttributes
+          - sns:CreateTopic
+          - sns:Subscribe
+          - sns:Unsubscribe
+          - sns:DeleteTopic
+          Resource: '*'
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: SNS
+        - Action:
+          - cloudwatch:PutDashboard
+          - cloudwatch:ListDashboards
+          - cloudwatch:DeleteDashboards
+          - cloudwatch:GetDashboard
+          Resource: '*'
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: CloudWatch
+        - Action:
+          - iam:PassRole
+          - iam:CreateRole
+          - iam:CreateServiceLinkedRole
+          - iam:DeleteRole
+          - iam:GetRole
+          - iam:TagRole
+          - iam:SimulatePrincipalPolicy
+          Resource: !Sub arn:${AWS::Partition}:iam::${AWS::AccountId}:role/*
+          Effect: Allow
+          Condition: !If
+            - IsMultiRegion
+            - StringEquals:
+                aws:RequestedRegion:
+                  - !Ref AWS::Region
+            - !Ref AWS::NoValue
+          Sid: IAMModify
+        - Action:
```

Comment:
Can you add a `# TODO: review and scope down IAM policies` comment next to the 3 IAM sections?